Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6612,6 +6599,166 @@ def create_wildfire_best_practice_profile_command(profile_name: str):
     return_results(f'The profile {profile_name} was created successfully.')
 
 
+def prettify_user_interface_config(interface_config: List) -> List:
+    pretty_interface_config = []
+    for interface in interface_config:
+        pretty_interface_config.append({
+            'Name': interface['@name'],
+            'Network': interface['network']
+        })
+
+    return pretty_interface_config
+
+
+def show_user_id_interface_config_request(args):
+    template = args.get('template') if args.get('template') else TEMPLATE
+    template_stack = args.get('template_stack')
+    vsys = demisto.params().get('vsys')
+    # firewall instance xpath
+    if VSYS:
+        xpath = "/config/devices/entry[@name='localhost.localdomain']/vsys/entry[@name=\'" + VSYS + "\']/zone"
+
+    # panorama instance xpath
+    elif not template_stack:
+        xpath = "/config/devices/entry[@name='localhost.localdomain']/template/entry[@name=\'" + template + "\']" \
+                "/config/devices/entry[@name='localhost.localdomain']/" \
+                "vsys/entry[@name=\'" + vsys + "\']/zone"
+    else:
```

Comment:
what is the else? `vsys` is usually for firewalls and you cover that in the beginning.

Reply:
So I checked with @jtschichold and it seems this limitation only exists in the XSOAR integration.
Virtual systems can be used on Panorama instances but if it is not in use the default is `vsys1` - hence why we added an argument to the command called vsys which defaults to the same argument which in turn will enable the command to run on Panorama.